### PR TITLE
[19.05] 19.05 Release Notes

### PR DESCRIPTION
**Pull request by jmchilton**
If you can just fix it instead of commenting, that'd be most appreciated - I'm quite exhausted from putting these together and I want nothing to do with them 😰🤣🙏. More and more automation with each release and somehow they are more work every time.

Despite saying I wasn't going to put together the user face release notes... I did actually outline those. If someone wants to go through and do better images and take a second pass at them though - that would be great (ping @erasche ?). The automated approach to bootstrapping tools and datatypes and vizs worked pretty well I think.

### CODE REVIEW

File: doc/source/releases/19.01_announce_user.rst [hunk 1]
```diff
@@ -41,13 +41,13 @@ There's a new discussion forum to replace Biostars. It is accessible under the '
    :alt: Galaxy's new help forum which replaces biostars
 
 
-New Visualisations
+New Vizualisations
```
Comment (martenson) [inline doc/source/releases/19.01_announce_user.rst]: s/Vizualisations/Visualizations

Comment (martenson) [reply doc/source/releases/19.01_announce_user.rst]: 🤣 just kidding, I will work on these next week ❤️

**Comment by hexylena** [comment]
> Despite saying I wasn't going to put together the user face release notes... I did actually outline those

John!

> If someone wants to go through and do better images and take a second pass at them though - that would be great (ping @erasche ?)

yea course, I volunteered for it after all :)

**Comment by hexylena** [comment]
@jmchilton done, did some language changes for the user focus + gifs for everything. Hopefully helps get users excited. Also: thanks! Gosh you did 90% of the work on these for me.

**Comment by martenson** [comment]
Marvelous!

**Pull request merged**